lib/runcible/models/puppet_install_distributor.rb

```diff
@@ -4,10 +4,11 @@
 module Runcible
   module Models
     class PuppetInstallDistributor < Distributor
-      attr_accessor 'install_path'
+      attr_accessor 'install_path', 'subdir'
 
-      def initialize(install_path, params = {})
+      def initialize(install_path, subdir = nil, params = {})
         @install_path = install_path
+        @subdir = subdir
         super(params)
       end
```

Comment:
any reason to not just use params for this?    iirc that is useful for non-required parameters such as this one.  It still requires hte attr_accessor to be added above but doesn't change the method signature (which would require a major version bump).
```
irb(main):003:0> Runcible::Models::PuppetInstallDistributor.new('/etc/puppet', :subdir => 'modules').config
=> {"install_path"=>"/etc/puppet", "auto_publish"=>false, "subdir"=>"modules"}
```

Reply:
Right, this is much cleaner! Thanks!